methods.py

```diff
@@ -1026,9 +1026,6 @@ def build_legacygl_header(filename, include, class_suffix, output_attribs):
             fd.write("\t\t\t{" + x["set_mask"] + "," + x["clear_mask"] + "},\n")
 
         fd.write("\t\t};\n\n")
-    else:
```

Comment:
At last! Those warnings have been there forever and are spammed at least 200 times during the build :D